redash/worker.py

```diff
@@ -79,3 +79,12 @@ def __call__(self, *args, **kwargs):
 def init_celery_flask_app(**kwargs):
     app = create_app()
     app.app_context().push()
+
+
+# Hook for extensions to add periodic tasks.
+@celery.on_after_configure.connect
+def add_periodic_tasks(sender, **kwargs):
+    app = create_app()
```

Comment:
There is no "risk" or cost in calling `create_app` again?

Reply:
For some context of why I added this `create_app()` here:

A function must be decorated with `@celery.on_after_configure.connect` in order for `add_periodic_task()` to work (e.g. http://docs.celeryproject.org/en/latest/userguide/periodic-tasks.html#entries) However, this event is triggered after a celery app has initialized but likely before workers have initialized. So it’s not possible to access the app created in a worker from this context. I would say the “cost” here is that we’re creating an additional app but I suspect it doesn’t interfere with the apps in the workers the way that the worker apps don’t interfere with each other.

Additionally, we’ve been running this code in the Mozilla fork for a couple of months and no issues have come up either.